tests/server/TestCSPConfig.js

```diff
@@ -21,19 +21,26 @@ const apiHosts = {
 
 
 describe('CSP Config Defaults', () => {
-  afterEach(() => {
-    process.env.NODE_ENV = 'production';
+  let oldNodeEnv;
+
+  before(() => {
+    oldNodeEnv = process.env.NODE_ENV;
+  });
+
+  after(() => {
+    process.env.NODE_ENV = oldNodeEnv;
   });
 
   // eslint-disable-next-line no-restricted-syntax
   for (const env of deployedEnvs) {
     it(`should have a source-list config for ${env}`, () => {
       process.env.NODE_ENV = env;
+      jest.resetModules();
       const cdnHost = cdnHosts[env];
       const apiHost = apiHosts[env];
-      const config = requireUncached('config');
+      const config = require('config');
       const cspConfig = config.get('CSP').directives;
-      assert.include(cspConfig.scriptSrc, cdnHost);
+      expect(cspConfig.scriptSrc).toContain(cdnHost);
```

Comment:
This was a test to see the better assertion failure messaging you get using expect - will need to migrate to that in due course.